elements/pfe-sass/extends/_typography_extends.scss

```diff
@@ -0,0 +1,55 @@
+/// EXTENDS
+/// @see {mixin} pfe-text 
+
+%pfe-content-base {
+    font-family: pfe-var(font-family);
+    line-height: pfe-var(line-height);
+    font-weight: pfe-var(font-weight--light);
+}
+
+/// CONTENT BLOCK
+%pfe-content--p {
```

Comment:
This doesn't really seem necessary since it's extending pfe-text and there aren't pfe-content--h* siblings.  It also seems to be a copy of pfe-text--md below.